### ISSUE TITLE
Ue4 builds create memory leak

### ISSUE THREAD

**Issue by ARtSSt** [microsoft/MixedReality-UXTools-Unreal#53]
Hello all, I am looking for help with a memory leak issue when we compile from unreal for hololens2.

We have narrowed down this issue to UE4 building to H2. We have stripped everything from our project and compile a app running nothing and it seems the memory is still slowly filling up.

We have scoured the Hololens2, the tmp directories and we cannot find wherever these files are written to. Clearing memory does not fix the issue.

We have tethered the device and run ever command line tool on earth to search the device memory. We can find nothing.

The only way to reclaim the space is to completely reformat the device with a fresh install of the OS.

Any one else see this behaviour or have any tips?

I am posting this in a few places to look for help.

**Comment by luis-valverde-ms** [comment]
Hey @ARtSSt , I am not clear from your description: are you running out of storage space in the HL2 device? I have seen that happen if you have accumulated too many crash dumps. You can check app crash dumps via the device portal: https://docs.microsoft.com/en-us/windows/mixed-reality/develop/advanced-concepts/using-the-windows-device-portal#app-crash-dumps/

**Comment by ARtSSt** [comment]
Yes we are running out of memory - but the files cannot be found causing the issue, it appears to be in temporary memory somewhere.  
This seems specific to builds from UE4 -  We tested this many ways and have stripped the app to only the basic XR plugins in a simple app build(which compiles fine) and deploy and we still see the slow memory leak. 

This is appears to be the same issue : 
https://docs.microsoft.com/en-us/answers/questions/732279/unable-to-delete-hololens-2-temporary-files-after.html

**Comment by luis-valverde-ms** [comment]
I see. So you are running out of disk/storage space, not RAM. @fieldsJacksonG , do you know who could be a good person to follow this up with?
